ReflectionAnnotationRegEx/src/main/java/com/onto/javacourse/displayer/Displayer.java

```diff
@@ -0,0 +1,76 @@
+package com.onto.javacourse.displayer;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Utility class to display details about the fields and methods of a given object's class.
+ * It uses reflection to retrieve information and logs it using SLF4J.
+ */
+public class Displayer {
+  private static final Logger LOGGER = LoggerFactory.getLogger(Displayer.class);
+
+  /**
+   * Logs details about the class of the given object, including its fields and methods.
+   *
+   * @param instance The object whose class details are to be logged.
+   */
+  public static void returnClassDetails(Object instance) {
```

Comment:
This is ok but won't it be more flexible if you extract the data and have it in an object? Not just print it. Because the current implementation won't be that useful.